addons/mail/static/src/core/common/chat_bubble.scss

```diff
@@ -0,0 +1,76 @@
+
+.o-mail-ChatBubble {
+    height: auto !important;
+    z-index: $o-mail-ChatBubble-zindex;
+    border: none !important;
+    background-color: transparent;
+    padding: 0;
+
+    &:hover {
+        .o-mail-ChatBubble-close {
+            visibility: visible;
+        }
+        .o-mail-ChatBubble-counter {
+            visibility: hidden;
```

Comment:
`display: none` should be used as much as possible, because `visibility` will hide the element to a screen reader. 
(see https://developer.mozilla.org/fr/docs/Web/CSS/visibility#accessibilit%C3%A9)